reana-ui/src/pages/profile/components/Quota.js

```diff
@@ -0,0 +1,68 @@
+/*
+  -*- coding: utf-8 -*-
+
+  This file is part of REANA.
+  Copyright (C) 2020 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+import React from "react";
+import { PieChart } from "react-minimal-pie-chart";
+import { Grid, Label } from "semantic-ui-react";
+
+import styles from "./Quota.module.scss";
+
+const SEPIA_COLOR = "#f5ecec";
+const DARK_SEPIA_COLOR = "#b68181";
+
+export default function Quota() {
+  const dummy = [
+    { title: "CPU", value: 71, limit: 120, unit: "min" },
+    { title: "HDD", value: 83232, limit: 123456, unit: "KiB" },
+  ];
+  return (
+    <Grid columns={2}>
+      {dummy.map(({ title, value, limit, unit }) => {
+        const percentage = ((value / limit) * 100).toFixed(2);
+        let percentageColor = "green";
+        // TODO: To be consistent with `reana-client quota-show`
```

Comment:
Maybe we could return a `quota_health` field from the server and do this logic there? If you think it makes sense we could add it as part of https://github.com/reanahub/reana-server/issues/289.

Note: this kind of goes a bit against the comment of doing the unit conversion client side... maybe I am leaning towards doing everything server side... Looking forward to reading your feedback.

Reply:
I agree, if it's going to be used both in `r-ui` and `r-client`, it makes sense to serve it from `r-server`.